test/e2e/serial/tests/configuration.go

```diff
@@ -1071,30 +1068,31 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 			pods, err := podlist.With(fxt.Client).ByDeployment(ctx, updatedDeployment)
 			Expect(err).ToNot(HaveOccurred())
 
-			// Here we have an assertion for the length of the pods to be smaller than the maxPodsWithTAE.
 			// Based on a couple of test runs it accurs that some pods sometimes may be created in cases that the time frame
 			// between the kubeletchanges and the deployment being created are too small for the scheduler to pick up the new changes and recover
-			// (sometimes even a second is too much) and it can create pods with Topology Affinity Error that have zero effect on the deployment itself
-			// and theirs status is ContainerStatusUnknown. the count should always be bounded and no more than the maxPodsWithTAE const.
-			Expect(len(pods)).To(BeNumerically("<", maxPodsWithTAE))
```

Comment:
Yes this is intended since the number of pods being created in the scenario is random and can be more than then maxPodsWithTAE. We don't know how many can be created before the pending pod comes. So there's no need to have this check because as long as there's exactly one pending pod it doesn't matter how many pods are failing since they have zero effect.

Reply:
I see your point, still, the number should be bounded. let's please soften this check and make it a warning instead  of completely dropping it.